### ISSUE TITLE
Token Action HUD does not display with new Token with a base class in PF1e

### ISSUE THREAD

**Issue by Golldan1111** [moo-man/WFRP4e-FoundryVTT#1158]
**Describe the bug**
Token Action HUD does not display with new PC Token with a base class in PF1e

**To Reproduce**
Create new PC, add Race and Class and drag to Scene and no Token Action HUD shows. If you Create new and add nothing or only Race, Token Action HUD displays

**Screenshots**

![image](https://user-images.githubusercontent.com/86072213/174456232-23bc512b-f70f-4134-ae50-d24362ee1a7e.png)
![image](https://user-images.githubusercontent.com/86072213/174456249-e4fe1d2a-3aa8-4bac-b8c1-ab49b87d3765.png)

**Version Numbers**
PLEASE PROVIDE YOUR FOUNDRY, SYSTEM, and MODULE VERSIONS (where applicable) 
9 - 269 - PF 0.81.0
![image](https://user-images.githubusercontent.com/86072213/174456287-4e0f0abd-5ae9-4a9c-967d-cab3b41ff85b.png)

Foundry:
wfrp4e:
wfrp4e-core:
wfrp4e-starter-set:
wfrp4e-rnhd:
wfrp4e-eis:
wfrp4e-ua1:
wfrp4e-dotr:
wfrp4e-archives1:
wfrp4e-middenheim:

**Comment by moo-man** [comment]
Did you post this in the wrong repo?

**Comment by Golldan1111** [comment]
Yeah sorry- just followed a link and wanted to get it posted. Plz close/delete.

Sent from Yahoo Mail on Android 
 
  On Sat, Jun 18, 2022 at 4:38 PM, ***@***.***> wrote:   


Did you post this in the wrong repo?
